crates/esthri-cli/tests/integration/http_server.rs

```diff
@@ -40,6 +41,88 @@ async fn test_fetch_object() {
     assert_eq!(body, "this file has contents\n");
 }
 
+#[tokio::test]
```

Comment:
Please add comment describing what the test aims to validate, including whether this addresses nominal behavior or edge cases if applicable.

Reply:
done